src/common/components/clusterConfiguration/networkConfigurationValidation.ts

```diff
@@ -34,7 +34,7 @@ export const getNetworkInitialValues = (
     hostSubnet: getSubnetFromMachineNetworkCidr(selectMachineNetworkCIDR(cluster)),
     vipDhcpAllocation: cluster.vipDhcpAllocation,
     managedNetworkingType: cluster.userManagedNetworking ? 'userManaged' : 'clusterManaged',
-    networkType: cluster.networkType || 'OpenShiftSDN',
+    networkType: cluster.networkType || 'OVNKubernetes',
```

Comment:
is the default supposed to be OVN for all clusters? or just SNO ones

Reply:
yes, that's the whole point of this task